pkg/validation/cluster/handler.go

```diff
@@ -95,3 +107,26 @@ func (h *AdmissionHandler) validateCreateOrUpdate(c *kubermaticv1.Cluster) error
 func (h *AdmissionHandler) SetupWebhookWithManager(mgr ctrl.Manager) {
 	mgr.GetWebhookServer().Register("/validate-kubermatic-k8s-io-cluster", &webhook.Admission{Handler: h})
 }
+
+func (h *AdmissionHandler) rejectEnableUserSSHKeyAgentUpdate(ctx context.Context, cluster *kubermaticv1.Cluster) error {
```

Comment:
This not just rejects Enablements, but also Disablements. The function could maybe be called `rejectUserSSHKeyAgentChanges`?